Pipfile

```diff
@@ -10,7 +10,7 @@ python_version = "3"
 # NuCypher
 umbral = "==0.1.3a2"
 constant-sorrow = ">=0.1.0a9"
-bytestring-splitter = ">=1.3.0"
+bytestring-splitter = ">=2.2.0"
```

Comment:
I am not able to push a relock of dependencies because when I do:
`make lock` in a docker container, my resulting requirements.txt files contain version specifications like:
```
+zope.interface==5.1.0; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4'
```
So, I'm not sure what to do about that.

Reply:
Still an unsolved mystery - although perhaps not an issue - it's unclear what impact this has.

Reply:
I spent a few hours on it yesterday.  Tried different docker images, ubuntu:latest, python3.7:latest.... they all add those version specs.

To be clear, this is _perfectly functional_

In any case, I will need someone else to generate dependencies for me :|

Reply:
> To be clear, this is perfectly functional

Yes - I see that it is working inside docker.  I can say the same for the deps in `main`

I don't see an Issue here - but I don't understand why dependencies are locked differently.  Especially since the python specifiers are completely irrelevant for nucypher.  

I'm baffled that OSX still ships with python 2.7 - a deprecated version of python :joy: - perhaps docker for mac is trying to accommodate that situation?

Some of the packages locked are not even required *whatsoever*, only for versions of python less than 3.6.